test/holding-marc-test.js

```diff
@@ -0,0 +1,131 @@
+/* global describe it */
+
+const assert = require('assert')
+const holdingSerializer = require('./../lib/serializers/holding')
+const HoldingSierraRecord = require('./../lib/models/holding-sierra-record')
+const Holding = require('./../lib/models/holding')
+const buildMapper = require('./../lib/field-mapper')
+
+/**
+ * Given an object (bib or item marc-in-json object)
+ *
+ * @return {object} Copy of given object with modified subfield content
+ function changeSubField (object, marcTag, subfieldTag, newContent) {
+   // Deep copy object:
+   object = JSON.parse(JSON.stringify(object))
+
+   object.varFields = object.varFields.map((f) => {
+     if (String(f.marcTag) === String(marcTag)) {
+       f.subFields = f.subFields.map((s) => {
+         if (String(s.tag) === String(subfieldTag)) {
+           s.content = newContent
+          }
+          return s
+        })
+      }
+      return f
+    })
+    return object
+  }
+ */
```

Comment:
Looks like a useful utility function for testing, but is it used?

Reply:
oh yeah, its not, I borrowed it from one of the other test classses because it seemed like I might need it, but I can't think of a test case where I would. I'll just remove it.